plugins/woocommerce-blocks/assets/js/settings/shared/default-fields.ts

```diff
@@ -63,21 +70,29 @@ export interface CoreAddress {
 	phone: string;
 }
 
-export type ShippingAddress = CoreAddress & Record< string, string | boolean >;
+export type AdditionalValues = Record<
```

Comment:
I don't think this works does it? I would expect the below to throw an error.

I still think we need "negated types" which are not available in TS yet.

<img width="234" alt="image" src="https://github.com/woocommerce/woocommerce/assets/5656702/01a95bf9-309b-4b3d-8892-ffab44e95dd7">